<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Quching</class>
 <widget class="QMainWindow" name="Quching">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="central_widget">
   <layout class="QHBoxLayout" name="central_widget_layout">
    <item>
     <widget class="QWidget" name="queue_widget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QVBoxLayout" name="queue_widget_layout">
       <item>
        <widget class="QWidget" name="queue_controls" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QToolButton" name="clear_button">
            <property name="toolTip">
             <string>clear queue</string>
            </property>
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset theme="QIcon::ThemeIcon::WindowClose"/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="repeat_button">
            <property name="toolTip">
             <string>repeat track</string>
            </property>
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset theme="QIcon::ThemeIcon::MediaPlaylistRepeat"/>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="shuffle_button">
            <property name="toolTip">
             <string>shuffle queue</string>
            </property>
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset theme="QIcon::ThemeIcon::MediaPlaylistShuffle"/>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QListView" name="queue_view">
         <property name="dragDropMode">
          <enum>QAbstractItemView::DragDropMode::InternalMove</enum>
         </property>
         <property name="movement">
          <enum>QListView::Movement::Snap</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QTabWidget" name="browser_tabs">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="artists_tab">
       <attribute name="title">
        <string>Artists</string>
       </attribute>
       <layout class="QVBoxLayout" name="artists_tab_layout">
        <item>
         <widget class="QStackedWidget" name="artists_stacked_widget">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="artists_page">
           <layout class="QVBoxLayout" name="artists_page_layout">
            <item>
             <widget class="QListView" name="artists_list">
              <property name="editTriggers">
               <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
              </property>
              <property name="iconSize">
               <size>
                <width>150</width>
                <height>150</height>
               </size>
              </property>
              <property name="gridSize">
               <size>
                <width>150</width>
                <height>150</height>
               </size>
              </property>
              <property name="viewMode">
               <enum>QListView::ViewMode::IconMode</enum>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="albums_page">
           <layout class="QVBoxLayout" name="albums_page_layout">
            <item>
             <widget class="QToolButton" name="back_button3">
              <property name="text">
               <string>...</string>
              </property>
              <property name="icon">
               <iconset theme="QIcon::ThemeIcon::GoPrevious"/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTreeWidget" name="albums_tree">
              <property name="iconSize">
               <size>
                <width>100</width>
                <height>100</height>
               </size>
              </property>
              <property name="headerHidden">
               <bool>true</bool>
              </property>
              <property name="columnCount">
               <number>3</number>
              </property>
              <column>
               <property name="text">
                <string notr="true">1</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string notr="true">2</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string notr="true">3</string>
               </property>
              </column>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="albums_tab">
       <attribute name="title">
        <string>Albums</string>
       </attribute>
       <layout class="QGridLayout" name="albums_tab_layout">
        <item row="1" column="0">
         <widget class="QStackedWidget" name="album_stacked_widgets">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="albums_list_page">
           <layout class="QVBoxLayout" name="albums_list_layout">
            <item>
             <widget class="QListView" name="albums_list">
              <property name="editTriggers">
               <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
              </property>
              <property name="iconSize">
               <size>
                <width>150</width>
                <height>150</height>
               </size>
              </property>
              <property name="gridSize">
               <size>
                <width>200</width>
                <height>200</height>
               </size>
              </property>
              <property name="viewMode">
               <enum>QListView::ViewMode::IconMode</enum>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tracklist_page">
           <layout class="QGridLayout" name="album_tracklists_layout">
            <item row="1" column="0" alignment="Qt::AlignmentFlag::AlignHCenter">
             <widget class="QLabel" name="cover_art">
              <property name="text">
               <string>coverArt</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QToolButton" name="back_button2">
              <property name="text">
               <string>...</string>
              </property>
              <property name="icon">
               <iconset theme="QIcon::ThemeIcon::GoPrevious"/>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QTreeWidget" name="album_tracks">
              <property name="columnCount">
               <number>2</number>
              </property>
              <column>
               <property name="text">
                <string>Title</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Duration</string>
               </property>
              </column>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="playlists_tab">
       <attribute name="title">
        <string>Playlists</string>
       </attribute>
       <layout class="QVBoxLayout" name="playlists_tab_layout">
        <item>
         <widget class="QStackedWidget" name="playlists_tab_widget">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="playlists_list_page">
           <layout class="QVBoxLayout" name="playlists_list_layout">
            <item>
             <widget class="QWidget" name="playlist_edit" native="true">
              <layout class="QHBoxLayout" name="playlist_edit_layout">
               <item>
                <widget class="QToolButton" name="play_playlist_button">
                 <property name="text">
                  <string>...</string>
                 </property>
                 <property name="icon">
                  <iconset theme="QIcon::ThemeIcon::MediaPlaybackStart"/>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QToolButton" name="new_playlist_button">
                 <property name="text">
                  <string>...</string>
                 </property>
                 <property name="icon">
                  <iconset theme="QIcon::ThemeIcon::DocumentNew"/>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QToolButton" name="edit_playlist_button">
                 <property name="text">
                  <string>...</string>
                 </property>
                 <property name="icon">
                  <iconset theme="QIcon::ThemeIcon::DocumentPageSetup"/>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QToolButton" name="delete_playlist_button">
                 <property name="text">
                  <string>...</string>
                 </property>
                 <property name="icon">
                  <iconset theme="QIcon::ThemeIcon::WindowClose"/>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="playlist_page_switcher">
                 <item>
                  <property name="text">
                   <string>Standard</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Dynamic</string>
                  </property>
                 </item>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer">
                 <property name="orientation">
                  <enum>Qt::Orientation::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QStackedWidget" name="stacked_playlists_lists">
              <property name="currentIndex">
               <number>1</number>
              </property>
              <widget class="QWidget" name="playlists_page">
               <layout class="QVBoxLayout" name="playlists_page_layout">
                <item>
                 <widget class="QListView" name="playlists_list">
                  <property name="editTriggers">
                   <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
                  </property>
                  <property name="iconSize">
                   <size>
                    <width>100</width>
                    <height>100</height>
                   </size>
                  </property>
                  <property name="movement">
                   <enum>QListView::Movement::Static</enum>
                  </property>
                  <property name="gridSize">
                   <size>
                    <width>150</width>
                    <height>150</height>
                   </size>
                  </property>
                  <property name="viewMode">
                   <enum>QListView::ViewMode::IconMode</enum>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
              <widget class="QWidget" name="dynamic_playlists_page">
               <layout class="QVBoxLayout" name="dynamic_playlists_layout">
                <item>
                 <widget class="QListView" name="dynamic_playlists_list">
                  <property name="iconSize">
                   <size>
                    <width>100</width>
                    <height>100</height>
                   </size>
                  </property>
                  <property name="gridSize">
                   <size>
                    <width>150</width>
                    <height>150</height>
                   </size>
                  </property>
                  <property name="viewMode">
                   <enum>QListView::ViewMode::IconMode</enum>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="playlist_view">
           <layout class="QVBoxLayout" name="playlist_view_layout">
            <item>
             <widget class="QToolButton" name="back_button">
              <property name="text">
               <string>...</string>
              </property>
              <property name="icon">
               <iconset theme="QIcon::ThemeIcon::GoPrevious"/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTreeWidget" name="playlist_tracks">
              <property name="iconSize">
               <size>
                <width>100</width>
                <height>100</height>
               </size>
              </property>
              <column>
               <property name="text">
                <string notr="true">Title</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Duration</string>
               </property>
              </column>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QDockWidget" name="controls_dock">
   <property name="features">
    <set>QDockWidget::DockWidgetFeature::DockWidgetMovable</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::DockWidgetArea::BottomDockWidgetArea|Qt::DockWidgetArea::TopDockWidgetArea</set>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="controls_dock_widget">
    <layout class="QGridLayout" name="gridLayout">
     <item row="3" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QToolButton" name="prev_button">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset theme="QIcon::ThemeIcon::MediaSkipBackward"/>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="play_button">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset theme="QIcon::ThemeIcon::MediaPlaybackStart"/>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignmentFlag::AlignLeft">
        <widget class="QToolButton" name="next_button">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset theme="QIcon::ThemeIcon::MediaSkipForward"/>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="curr_time">
         <property name="text">
          <string>0:00</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="seek_slider">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="total_time">
         <property name="text">
          <string>0:00</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="7" column="0" alignment="Qt::AlignmentFlag::AlignRight">
      <widget class="QLabel" name="volume_percent">
       <property name="text">
        <string>0%</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item alignment="Qt::AlignmentFlag::AlignLeft">
        <widget class="QLabel" name="artist_label">
         <property name="text">
          <string>%artist%</string>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignmentFlag::AlignLeft">
        <widget class="QLabel" name="title_label">
         <property name="text">
          <string>%title%</string>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignmentFlag::AlignLeft">
        <widget class="QLabel" name="album_label">
         <property name="text">
          <string>%album%</string>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignmentFlag::AlignLeft">
        <widget class="QLabel" name="track_label">
         <property name="text">
          <string>0/0</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="6" column="0" alignment="Qt::AlignmentFlag::AlignRight">
      <widget class="QSlider" name="volume_slider">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
